tests/utils/fixture.ts

```diff
@@ -191,6 +192,20 @@ export async function runPluginStep(
   constants: Partial<NetlifyPluginConstants> = {},
 ) {
   const stepFunction = (await import('../../src/index.js'))[step]
+
+  let netlifyConfig = {
+    headers: [],
+    redirects: [],
+  }
+
+  // load netlify.toml if it exists
+  if (existsSync(join(ctx.cwd, 'netlify.toml'))) {
+    const resolvedNetlifyConfig = await resolveNetlifyConfig({ cwd: ctx.cwd })
+    if (resolvedNetlifyConfig.config) {
+      netlifyConfig = resolvedNetlifyConfig.config
+    }
+  }
```

Comment:
added this to support one of added test route handlers that reads non-nextjs produced html file and for that to work, we need to pass netlifyConfig produced from `netlify.toml` which instructs bundling to include some external files